src/pages/AddNewBooksForm/AddNewBooksForm.tsx

```diff
@@ -47,6 +47,10 @@ const AddNewBooksForm = () => {
   const [amount, setAmount] = useState(1)
   const [description, setDescription] = useState('')
   const [imageUrl, setImageUrl] = useState('')
+  const [titleError, setTitleError] = useState(false)
+  const [authorError, setAuthorError] = useState(false)
+  const [amountError, setAmountError] = useState(false)
+  const [descriptionError, setDescriptionError] = useState(false)
```

Comment:
Do we need individual state for each error. If you use just hasError state you could achieve same effect?

Reply:
I checked, it is doable by creating useState array `const [errors, setErrors] = useState([])` which would hold types of errors and then we could map this error array and bind error types to their elements. This solution requires time and effort that we should not provide at this time so resolution is: Won't do.